Scripts/eas-realtime-probe.c

```
@@ -0,0 +1,99 @@
+    EAS_PCM *pcm;
+    long frames = 0;
+    int peak = 0, i, s;
+    struct timespec t0, t1;
+    double wall, seconds;
+
+    if (argc < 2) { fprintf(stderr, "usage: %s <midi>\n", argv[0]); return 2; }
+    f = fopen(argv[1], "rb");
+    if (!f) { fprintf(stderr, "cannot open %s\n", argv[1]); return 2; }
+    fseek(f, 0, SEEK_END); len = ftell(f); fseek(f, 0, SEEK_SET);
+    buf = malloc(len);
+    if (fread(buf, 1, len, f) != (size_t)len) { fprintf(stderr, "short read\n"); return 2; }
```

Cppcheck (2.21.0)

[error] Resource leak

(resourceLeak)